Signal/src/call/CallService.swift

```diff
@@ -327,8 +327,9 @@ protocol CallServiceObserver: class {
 
             // Don't let the outgoing call ring forever. We don't support inbound ringing forever anyway.
             let timeout: Promise<Void> = after(interval: connectingTimeoutSeconds).then { () -> Void in
-                // rejecting a promise by throwing is safely a no-op if the promise has already been fulfilled
-                OWSProdInfo(OWSAnalyticsEvents.callServiceErrorTimeoutWhileConnectingOutgoing(), file: #file, function: #function, line: #line)
```

Comment:
As a separate change, I've moved this logging so it only occurs when relevant and updated the comment.